ON-CALL.md

```diff
@@ -0,0 +1,60 @@
+# Istio On-call Playbook
+
+## Who
+Every week, one Istio developer is on-call and is responsible for maintaining Istio build and for helping out users and other developers. The on-call person should prioritize on-call duties on top of the daily work.
+
+## Schedule
```

Comment:
And how do we do this oncall rotation? Based on volunteer or we have some rotation mechanism like round robin

Reply:
I do not know this.